src/internal/js/mod.rs

```diff
@@ -307,10 +313,26 @@ impl JsString {
         }
     }
 
+    pub fn ucs2_value(self) -> Vec<u16> {
```

Comment:
I guess the overhead mostly comes from processing UTF-8 strings against JS API which works on UCS-2 codepoints (this is why V8 uses UTF-8 internally rarely). Using such strings just for storing & console.log-ging should be fine though, even for small strings.

Also, we could try and allocate one-byte-kind strings for anything that is latin-1 string on Rust side, in which case it's also just a copy in the worst case with no need for transform.